app/assets/javascripts/discourse/app/controllers/edit-slow-mode.js

```diff
@@ -13,7 +13,7 @@ export default Controller.extend(ModalFunctionality, {
   hours: null,
   minutes: null,
   seconds: null,
-  saveDisabled: false,
+  submittingDisabled: false,
```

Comment:
@jjaffeux I'm not sure about this naming, does this look good to you? 

The reason I've changed it is that there is no Save button on the modal dialog. This property is used to
disable submitting during a request to the server.